### DISCUSSION TITLE
Create folders in launcher

### DISCUSSION THREAD

**Discussion by D8ma** [linuxdeepin/developer-center#3281] [Features Request & Ideas | 特性请求 & 头脑风暴]
![gnome-shell-activities-overlay (1).jpg](https://user-images.githubusercontent.com/107427853/176640430-ab82a534-1062-43da-8281-67cc1a2edf98.jpg)![deepin-v20-apps.jpg](https://user-images.githubusercontent.com/107427853/176640441-01734534-93fc-4753-a799-190b54336e4f.jpg)

**Comment by meiyixiang** [comment]
你好在V23我们会支持你说的启动器自定义分类功能。
